app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt

```diff
@@ -274,7 +275,15 @@ class EditPatientScreen(context: Context, attributeSet: AttributeSet) : Relative
   }
 
   override fun showColonyOrVillagesList(colonyOrVillageList: List<String>) {
+    ArrayAdapter<String>(context, android.R.layout.simple_list_item_1, colonyOrVillageList).also { adapter ->
+      colonyOrVillageEditText.setAdapter(adapter)
+    }
 
+    colonyOrVillageEditText.threshold = 0
```

Comment:
This doesn't make any difference, by default Android applies a threshold of 1 when the threshold is 0. So we can remove this.

https://developer.android.com/reference/android/widget/AutoCompleteTextView?authuser=1#setThreshold(int)